packages/icons/.svgrrc.cjs

```diff
@@ -0,0 +1,36 @@
+// See https://react-svgr.com/docs/options/
+
+module.exports = {
+  icon: true,
+  typescript: true,
+  expandProps: true,
+  removeDimensions: false,
+  replaceAttrValues: {
+    "#000": "currentColor"
```

Comment:
Would this also match `#000000`?

Reply:
Good catch, I suppose I should add 'black' as well.

Reply:
Nein! 🚢 😂